docs/apps/seqtk.md

```diff
@@ -0,0 +1,63 @@
+
+# Seqtk
+
+## Description
+
+Seqtk is a fast and lightweight tool for processing sequences in the FASTA or FASTQ format. It seamlessly parses both FASTA and FASTQ files which can also be optionally compressed by gzip.
+
+
+llumina BaseSpace command line client,`bs`, can be used to retrievie data from the Illumina BaseSpace environment to Puhti.
```

Comment:
Is this by design here?, it's the exact same as in the bs.md file. Perhaps a link to the bs.md file would be better if we want to refer to it?